deploy/003-deploy-VBep20.ts

```diff
@@ -59,8 +75,27 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
     autoMine: true,
   });
+
+  await deploy("vBNB", {
+    contract: "VBep20Immutable",
+    from: deployer,
+    args: [
+      wbnbDeployment.address,
+      comptrollerDeployment.address,
+      interestRateModelVUSDCDeployment.address,
+      "1500000000000000000",
```

Comment:
After refactoring the scripts to work on live networks I don't think we should define token configs like this. I think it depends on how this would be used to know how it is best defined but it should probably be included in the script deploying the contract or initializing it. After the contract is deployed we can use the `deployments` to fetch it so we should only need a config like this once.